help/destinations/catalog/voice/medallia-connector.md

```diff
@@ -0,0 +1,108 @@
+
+
+## Export type and frequency {#export-type-frequency}
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | You are exporting all newly qualified members of a segment, together with the desired schema fields (for example: email address, phone number, last name), as chosen in the select profile attributes screen of the [destination activation workflow](/help/destinations/ui/activate-batch-profile-destinations.md#select-attributes).|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+> 
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/connect-destination.html). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+* **OAuth Token Endpoint URL**: Typically takes the form of https://instance.medallia.tld/oauth/tenant/token
+* **Client ID**: Obtain from your Medallia delivery team
+* **API Gateway URL**: Typically takes the form of https://instance-tenant.apis.medallia.com
+* **Client Secret**: Obtain from your Medallia delivery team
+
+<img src="/help/destinations/assets/catalog/voice/medallia-destination-oauth.png" width="60%">
+
+
+### Fill in destination details {#destination-details}
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Import API Name]**: Name of the Medallia Import API (also known as Web Feed) to be used in this connection. Youcan activate different segments to different Import APIs to trigger different survey programs
```

Comment:
Minor comment: You can (missing space). And please add a period at the end of the sentence.

Reply:
Thanks for catching! Updated